site/content/docs/5.3/forms/range.md

```diff
@@ -50,6 +50,28 @@ By default, range inputs "snap" to integer values. To change this, you can speci
 <input type="range" class="form-range" min="0" max="5" step="0.5" id="customRange3">
 {{< /example >}}
 
+## Usability
+
+For better usability, it is recommended most of the time to display the current selected value.
+
+This requires extra JavaScript code.
+
+{{< example >}}
+<div class="d-flex w-100 justify-content-between">
+  <label for="customRange4" class="form-label">Example range</label>
+  <output id="customRange4ValueText" for="customRange4" class="fw-bold" aria-hidden="true"></output>
+</div>
+<input type="range" class="form-range" min="0" max="100" step="1" id="customRange4" onchange="updateLabelValue()">
```

Comment:
Should we have `onchange` or maybe just add a listener on it (since we introduce Js anyway)?

Reply:
Sorry but I don't understand the comment 🤔 
You want to change the `onchange` by something else?

Reply:
Interesting idea! I'll try to dig a bit to check if we can do easily something more reusable with a loop and id management.

Reply:
@louismaximepiton What do you think about the following?

```html
<!-- To test with another use case in the range page -->
<div class="d-flex w-100 justify-content-between mt-5">
  <label for="customRange5" class="form-label">Example range</label>
  <output id="customRange5ValueText" for="customRange5" class="fw-bold" aria-hidden="true"></output>
</div>
<input type="range" class="form-range" min="0" max="100" step="1" id="customRange5" onchange="updateLabelValue()">

<script>
  function updateLabelValue() {
    if (this.id === undefined) {
      return
    }

    document.getElementById(this.id + "ValueText").innerHTML = document.getElementById(this.id).value;
  }

  window.addEventListener('load', function () {
    Array.from(document.getElementsByClassName('form-range')).forEach(function (el) {
      if (document.getElementById(el.id + "ValueText")) {
        el.addEventListener('input', updateLabelValue)
        updateLabelValue.call(el)
      }
    })
  })
</script>
```

Reply:
> Remove the onchange call in the html but seems to be the case with your js

I'm dumb, forgot to remove the `onchange` in the HTML. That's why I had double calls and needed to add the `if (this.id === undefined) {` 🤦 

> Remove the dependency to a specific id value

Like your approach by forcing the `output` usage.

Just added a comment on top of the `<script>` to explain that it can be modified depending on the needs

So fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1714/commits/735a0a9836732acb6234226233d2ad68e8906817